packages/server/manage_container/client/manage_to_db/client.go

```diff
@@ -326,3 +328,19 @@ func (c Client) GetMatchingColumn(dataID string) (int32, error) {
 
 	return data.Meta.MatchingColumn, nil
 }
+
+func (c Client) CreateStatusFile(jobUUID string) {
+	ls.Lock(jobUUID)
+	defer ls.Unlock(jobUUID)
+	path := fmt.Sprintf("%s/%s", resultDbPath, jobUUID)
+	os.Mkdir(path, 0777)
+	fp, _ := os.Create(fmt.Sprintf("%s/status_%s", path, pb_types.JobStatus(2).String()))
+	fp.Close()
+}
```

Comment:
Use that return value as it may return an error.

- https://pkg.go.dev/os#Create
- https://pkg.go.dev/os#Mkdir

```suggestion
	err := os.Mkdir(path, 0777)
	if err != nil {
		return err
	}
	fp, err := os.Create(fmt.Sprintf("%s/status_%s", path, pb_types.JobStatus(2).String()))
	fp.Close()
	return err
}
```

Reply:
@arukuka 
I fix it with [b03a423](https://github.com/acompany-develop/QuickMPC/pull/150/commits/b03a423a475c97603500751f4d97f65fb8f53ec7)